tardis/plasma/properties/partition_function.py

```diff
@@ -190,7 +190,7 @@ def _main_nlte_calculation(
                     else:
                         raise e
                 general_level_boltzmann_factor[i].ix[species] = \
-                    level_boltzmann_factor * g[species][0] / level_boltzmann_factor[0]
+                    level_boltzmann_factor * g.loc[species][0] / level_boltzmann_factor[0]
```

Comment:
`.loc[species, 0]`

Reply:
That doesn't work...

Reply:
hmm  - that's interesting. I'll check it tomorrow.

Reply:
yes you're right - I screwed things up in my mind. I think the way you do it now is good.

Reply:
Remember, for example species=(14,1) for Si II